jsonschema-core.xml

```diff
@@ -812,22 +812,26 @@
                     location, annotation keywords need to specify any unusual handling of
                     multiple applicable occurrences of the keyword with different values.
                 </t>
-                <t>
-                    The default behavior is simply to collect all values in a list in
-                    indeterminate order.  Given the extensibility of keywords, including
-                    applicators, it is not possible to define a universally predictable
-                    order of processing.
```

Comment:
Seems like the spec is moving towards defining rules for handling annotations in overlapping subschemas (as this paragraph is removed, and the wording in `readOnly` and `writeOnly` show), which is good.